python/lsst/ci/hsc/gen2/gen3.py

```diff
@@ -27,13 +27,13 @@
 from lsst.daf.butler import Butler, ButlerConfig, Registry, Datastore, Config, StorageClassFactory
 from lsst.daf.butler.gen2convert import ConversionWalker, ConversionWriter
 
-REPO_ROOT = os.path.join(getPackageDir("ci_hsc"), "DATA")
+REPO_ROOT = os.path.join(getPackageDir("ci_hsc_gen2"), "DATA")
 
 converterConfig = Config(os.path.join(getPackageDir("daf_butler"), "config/gen2convert.yaml"))
 converterConfig["skymaps"] = {"ci_hsc": os.path.join(REPO_ROOT, "rerun", "ci_hsc")}
 converterConfig["regions"][0]["collection"] = "shared/ci_hsc"
 
-searchPaths = [os.path.join(getPackageDir("ci_hsc"), "gen3config"), ]
+searchPaths = [os.path.join(getPackageDir("ci_hsc_gen2"), "gen3config"), ]
 
 
 class Gen3ButlerWrapper:
```

Comment:
Please add docstrings to all the methods in here

Reply:
Much of this code is intended to be internal to this package. As this package is not intended to live past November, it is probably not worth documenting these methods, or making many other changes not needed to support its current use. If any of this functionality gets moved out of this package there will be a ticket to properly update everything to current standards.